test/test-file.cpp

```diff
@@ -46,40 +47,51 @@ namespace quick_lint_js {
 namespace {
 filesystem::path make_temporary_directory();
 void write_file(filesystem::path, const std::string &content);
+std::optional<filesystem::path> temp_directory_path;
```

Comment:
Blocking: This shouldn't be a global variable. Put this variable in the test_file class.